http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpTransport.scala

```diff
@@ -101,13 +103,25 @@ trait HttpTransport
     with PostHttpTransport {}
 
 trait CoreGet {
+  def GET[A](
+    url: URL,
+    headers: Seq[(String, String)] = Seq.empty)(implicit rds: HttpReads[A], hc: HeaderCarrier, ec: ExecutionContext): Future[A]
+
   def GET[A](
     url: String,
     queryParams: Seq[(String, String)],
     headers: Seq[(String, String)])(
       implicit rds: HttpReads[A],
       hc: HeaderCarrier,
-      ec: ExecutionContext): Future[A]
+      ec: ExecutionContext): Future[A] = {
```

Comment:
This is to ensure we don't introduce a deviation from existing behaviour. This was initially present in HttpGet